include/envoy/http/filter.h

```diff
@@ -86,6 +86,14 @@ enum class FilterTrailersStatus {
   StopIteration
 };
 
+/**
+ * Return codes for encode metadata filter invocations. Metadata should not stop filter iteration.
```

Comment:
Maybe change to "Metadata currently can not stop filter iteration" since it's a bit off to have a return code with one return :-)
I prefer to keep the return so if we do one day add it we don't have to refactor all the filters.

Reply:
Fixed. Thanks!